src/panoramic/cli/errors.py

```diff
@@ -348,7 +348,7 @@ class MissingFieldFileError(ValidationError):
     dataset_slug: str
     data_source: str
     data_reference: str
-    identifier: bool
+    identifier: bool  # FIXME: might not be used anymore
```

Comment:
I think this attribute was used only to further influence the field_type - I think we might want to drop it altogether.